packages/editor-ui/src/components/mixins/nodeHelpers.ts

```diff
@@ -44,8 +43,8 @@ export const nodeHelpers = mixins(
 			...mapGetters('credentials', [ 'getCredentialTypeByName', 'getCredentialsByType' ]),
 		},
 		methods: {
-			isHttpRequestNodeV2 (node: INodeUi): boolean {
-				return node.type === HTTP_REQUEST_NODE_TYPE && node.typeVersion === 2;
+			hasProxyAuth (node: INodeUi): boolean {
+				return node.parameters.nodeCredentialType !== undefined;
```

Comment:
on second thought, we should not be checking the value here. we should be checking the list of params if it contains `nodeCredentialType` param.. should not matter the name of the param here.